crates/pallets/starknet/src/tests/call_contract.rs

```diff
@@ -90,3 +91,27 @@ fn given_call_contract_call_works() {
         pretty_assertions::assert_eq!(res.unwrap(), vec![Felt252Wrapper::from_hex_be("0x02").unwrap()]);
     });
 }
+
+#[test]
+fn given_call_contract_call_works_jhnn() {
```

Comment:
why adding this test in `call_contract` ? I don't think that it emits events

Reply:
this was me building understanding that i accurately seeded the mock with a contract